earth_enterprise/src/third_party/gdal/SConscript

```diff
@@ -26,11 +26,26 @@
 # lines below.
 
 Import('third_party_env')
+import os
 
 gdal_version = 'gdal-2.1.2'
 openjpeg_version = 'openjpeg-v2.1.2-linux-x86_64'
 # kakadu_version = 'v5_2_6-00418C'
 # geoexpress_version = 'MrSID_DSDK-8.5.0.3422'
+
+# Check if MrSID decoding library is installed and get the decoding SDK path
+mrSIDLibName = 'libltidsdk.so'
+mrSIDDSDKPath = ""
+findLibraryCommand = "/sbin/ldconfig -p | grep %s | tr ' ' '\n' | grep /" %mrSIDLibName
```

Comment:
There is also a "scons" way to check for the library. If you take a look at the [gTest Sconscript](https://github.com/google/earthenterprise/blob/master/earth_enterprise/src/third_party/gtest/SConscript), you'll see what I mean. This works similarly to the "configure" step in building with make. It has the added benefit of not just checking ldconfig, but also verifies that it can be loaded with the compiler that scons is configured to use. But, I'll leave it up to you whether you think it's worth the trouble to convert to that method.

Reply:
hmm.. you mean:
conf = Configure(env)
if not conf.CheckLib('gtest'):
?
I don't see a way from there to get the path of the lib.